### PR TITLE
A few refactors in some files

### PR DESCRIPTION
**Pull request by sadra-barikbin**
Hi there!

A few tiny refactors in some files.

### CODE REVIEW

File: src/textord/tordmain.cpp [hunk 1]
```diff
@@ -533,16 +533,17 @@ bool Textord::clean_noise_from_row( // remove empties
       }
     }
   }
+  bool rejected = dot_count > norm_count * textord_noise_normratio && \
+                  dot_count > 2 && \
+                  super_norm_count < textord_noise_sncount;
```
Comment (stweil) [inline src/textord/tordmain.cpp:538]:
```suggestion
  bool rejected = dot_count > norm_count * textord_noise_normratio &&
                  dot_count > 2 &&
                  super_norm_count < textord_noise_sncount;
```

File: src/textord/tordmain.cpp [hunk 2]
```diff
@@ -533,16 +533,17 @@ bool Textord::clean_noise_from_row( // remove empties
...
   if (textord_noise_debug) {
     tprintf("Row ending at (%d,%g):", blob_box.right(), row->base_line(blob_box.right()));
     tprintf(" R=%g, dc=%d, nc=%d, %s\n",
             norm_count > 0 ? static_cast<float>(dot_count) / norm_count : 9999, dot_count,
             norm_count,
-            dot_count > norm_count * textord_noise_normratio && dot_count > 2 ? "REJECTED"
-                                                                              : "ACCEPTED");
+            rejected? "REJECTED": "ACCEPTED");
```
Comment (stweil) [inline src/textord/tordmain.cpp:544]: The original code did not use `super_norm_count < textord_noise_sncount` here.

Comment (sadra-barikbin) [reply src/textord/tordmain.cpp:544]: But it includes the expression in the return statement.

Comment (sadra-barikbin) [reply src/textord/tordmain.cpp:544]:
```python
return super_norm_count < textord_noise_sncount &&
         dot_count > norm_count * textord_noise_rowratio && dot_count > 2;
```

Comment (stweil) [reply src/textord/tordmain.cpp:544]: Yes, the debug message and the return statement used different expressions in the past. I don't know whether that was intentional or a bug. The new code uses the same expression for both locations.

**Review by stweil — CHANGES_REQUESTED** [review]
With my two suggested changes old and new code should be equivalent and we are safe. Maybe the original patch would be better, but it might change the output of the debug tprintf, and I am not sure about that.

**Comment by egorpugin** [comment]
Just revert and dont touch that condition.

**Comment by stweil** [comment]
Often it is easier if unrelated changes like in this pull request are split in separate pull requests, for example "Remove unused local variable", "Fix typo in comment", "Simplify code by using BlobGridSearch" and so on. Then those contributions are easy to review, and they can be merged quickly.

**Comment by sadra-barikbin** [comment]
I made PRs #4226 , #4227 , #4228 and #4229 for the other changes.

**Review by stweil — APPROVED** [review]
Thank you.

**Pull request merged**